charts/m4d/templates/m4d-config.yaml

```diff
@@ -20,9 +20,6 @@ data:
   VAULT_USER_MOUNT: {{ printf "/v1/sys/mounts/m4d/user-creds" | quote }} # temporary
```

Comment:
User credentials are not registered in vault. Can be removed.
Dataset credentials mount is not needed either.

Reply:
Dataset credentials mount is still used for configuration that does not use Vault plugins to retrieve the credentials such as Egeria catalog (soon to be addressed). So Dataset credentials mount is still needed.
I will remove VAULT_USER_MOUNT. Thanks